src/aiori-IME.c

```diff
@@ -0,0 +1,277 @@
+/* -*- mode: c; c-basic-offset: 8; indent-tabs-mode: nil; -*-
+ * vim:expandtab:shiftwidth=8:tabstop=8:
+ */
+/******************************************************************************\
+*                                                                              *
+*        Copyright (c) 2003, The Regents of the University of California       *
```

Comment:
Could you please add the DDN copyright below line 6:
Copyright (C) 2018, DataDirect Networks

Reply:
Done